src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/domain/makerecalldecisions/recommendation/RecommendationResponse.kt

```diff
@@ -73,6 +74,48 @@ data class PersonOnProbation(
   val primaryLanguage: String? = null,
   val hasBeenReviewed: Boolean? = false
 )
+fun PersonOnProbation.toPersOnProbationDto(): PersonOnProbationDto {
```

Comment:
Spelling mistake, should be lower case O

Reply:
Corrected :)